Word-to-PDF-Conversion/Convert-Word-document-to-PDF/Blazor/Server-app/Convert-Word-Document-to-PDF/Pages/DocIO.razor

```diff
@@ -0,0 +1,21 @@
+﻿@page "/docio"
+@using System.IO;
+@using Convert_Word_Document_to_PDF;
+@inject Convert_Word_Document_to_PDF.Data.WordService service
+@inject Microsoft.JSInterop.IJSRuntime JS
+
+<h2>Syncfusion DocIO library (Essential DocIO)</h2>
+<p>Syncfusion DocIO library (Essential DocIO) is a Blazor DocIO library used to create, read, edit, and convert Word files in your applications without Microsoft Office dependencies.</p>

```

Comment:
Syncfusion Word (DocIO) library is used to create, read, edit, and convert Word files in your applications without Microsoft Office dependencies.

Reply:
Updated sir